frontend/src/app/register/register.ctrl.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Controller for register page
+ */
+angular.module('flocs.user')
+.controller('registerCtrl', ['$scope','$location','$log','userDao', 
+    function($scope,$location,$log,userDao){
+
+	    function send(){
+            var username = $scope['username'];
+            $log.log($scope['username']);
+            var firstname = $scope['firstname'];
+            var lastname = $scope['lastname'];
+            var email = $scope['email'];
+			var passwd = $scope['password'];
+			if (userDao.registerUser(username,firstname,lastname,email,passwd) === true){
+				$location.url("/succes.html");
```

Comment:
Sorry could you explain this. I don't understand what is ment by states and
hardcoded urls. Which function is supposed to be used?

On Mon, Nov 23, 2015 at 3:16 PM, Jiří Mauritz <notifications@github.com>
wrote:

> In frontend/src/app/register/register.ctrl.js
> <https://github.com/effa/flocs/pull/56#discussion_r45606686>:
>
> > +/*
> > + * Controller for register page
> > + */
> > +angular.module('flocs.user')
> > +.controller('registerCtrl', ['$scope','$location','$log','userDao',
> > +    function($scope,$location,$log,userDao){
> > +
> > +	    function send(){
> > +            var username = $scope['username'];
> > +            $log.log($scope['username']);
> > +            var firstname = $scope['firstname'];
> > +            var lastname = $scope['lastname'];
> > +            var email = $scope['email'];
> > +			var passwd = $scope['password'];
> > +			if (userDao.registerUser(username,firstname,lastname,email,passwd) === true){
> > +				$location.url("/succes.html");
>
> do not use $location.url() - you navigate to the specific url, but you
> should go to the states declared in app.mdl.js (there are no hard coded
> urls)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/effa/flocs/pull/56/files#r45606686>.
>



-- 
Best regards,

Jakub Peschel